pkg/api/types.go

```diff
@@ -1763,7 +1767,9 @@ type ServiceSpec struct {
 	// This field will be ignored if the cloud-provider does not support the feature.
 	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
 
-	// Optional: Supports "ClientIP" and "None".  Used to maintain session affinity.
+	// Optional, used to maintain session affinity.
+	// Supports "ClientIP", "RequireNodeLocal" and "None". The second makes
+	// most sense with `Type: ClusterIP`. Defaults to None.
```

Comment:
Done. Won't we need to support RequireNodeLocal and AlphaRequireNodeLocal for at least one release, to avoid disruptions after a cluster upgrade?